src/commands/sarif/upload.ts

```diff
@@ -210,6 +184,15 @@ export class UploadSarifReportCommand extends Command {
         return false
       }
 
+      const potentialErrors = checkForError(sarifReport)
+      if (potentialErrors.length > 0) {
+        for (const potentialError of potentialErrors) {
+          this.context.stdout.write(renderInvalidFile(sarifReport, potentialError))
```

Comment:
Please review the `renderInvalidFile` function.

It should be called once or else we're going to be repeating things. We should change it to accept a `[]string` instead and then loop over the errors here (https://github.com/DataDog/datadog-ci/blob/1e909b1b537a143a749b3f808dcb1b2010c21ba5/src/commands/sarif/renderer.ts#L22)